packages/render-fidelity-tools/src/workflows/test-fidelity.ts

```diff
@@ -43,6 +44,7 @@ try {
 
 let scenarioWhitelist: Set<string>|null = null;
 
+// add additional arguments into scenariowhitelist which will not be rendered
```

Comment:
These arguments are the whitelist, which means only these scenarios will be rendered. A blacklist would be a list of scenarios to not render.